package.json

```diff
@@ -10,6 +10,7 @@
   },
   "devDependencies": {
     "@types/chromecast-caf-receiver": "^5.0.9",
+    "bitmovin-player": "^8.56.0-b.3",
```

Comment:
FYI: I will merge this PR once `8.56.0` is released and I can use it here as a dev dependency. I also did the manual interaction release tests for CAF using this receiver.